static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -191,6 +193,11 @@ define([
         // initialise email/outbrain check dispatcher
         checkDispatcher.init();
 
+        // initialise tailor overlay survey
+        if (config.switches.tailorSurveyOverlay) {
+            tailorSurveyOverlay.init();
```

Comment:
We try and bootstrap everything with the bootstrapContext, which will wrap your module in error reporting. I **think** as you have an init function you should just be able to do `bootstrapContext('tailorSurveyOverlay', tailorSurveyOverlay);`